README.md

```diff
@@ -77,4 +77,4 @@ as a `provided` dependency.
     <version>22.02.0-SNAPSHOT</version>
     <scope>provided</scope>
 </dependency>
-```
\ No newline at end of file
+```
```

Comment:
I'm not sure why this is a diff and why we need it.

Reply:
Looks like the previous version of the file did not end with a `\n` character, and this one does.  Not sure why it would not have?  I can remove it if people prefer, or leave it as is and merge. 

Related: https://thoughtbot.com/blog/no-newline-at-end-of-file